### PR TITLE
update faq vac cert invalid

### PR DESCRIPTION
**Pull request by dsarkar**

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -135,10 +135,10 @@
...
-                            "It has not yet been conclusively clarified who exactly, in addition to the vaccination centers, will be able to issue a vaccination certificate on the basis of proof of vaccination, i.e. the yellow vaccination booklet or the above-mentioned vaccination proof. The network will be gradually expanded."
+                            "There are currently (as of 10 June 2021) only a few places from which certificates with QR codes are issued, which can be scanned with the app.",
+                            "Usually, any document until today is not yet a valid digital vaccination certificate, but just a vaccination documentation according to §22 of the German Infection Protection Act, that is incompatible to the latest aligned European guidelines for a digital vaccination certificate. Therefore, this document, as well as the barcodes and QR codes in the yellow vaccination booklet, <b>cannot</b> be scanned with the Corona-Warn-App.",
```
Comment (MikeMcC399) [inline src/data/faq.json:139]: "incompatible to" =>
"incompatible with"
https://dictionary.cambridge.org/dictionary/english/incompatible

File: src/data/faq.json [hunk 2]
```diff
@@ -135,10 +135,10 @@
...
+                            "Usually, any document until today is not yet a valid digital vaccination certificate, but just a vaccination documentation according to §22 of the German Infection Protection Act, that is incompatible to the latest aligned European guidelines for a digital vaccination certificate. Therefore, this document, as well as the barcodes and QR codes in the yellow vaccination booklet, <b>cannot</b> be scanned with the Corona-Warn-App.",
+                            "The new digital certificate statements (that can be scanned) become available by authorized agencies only starting on 14th of June, leveraging a central service implemented on behalf of the federal government. The field test for this has been completed, and the feature is now being rolled out nationwide. More information on possible local sites can be found here: <a href='https://www.aponet.de/artikel/corona-digitaler-impfnachweis-ab-14-juni-in-apotheken-24262' target='_blank' rel='noopener noreferrer'>Corona: Digitaler Impfnachweis ab 14. Juni in Apotheken</a>.",
```
Comment (MikeMcC399) [inline src/data/faq.json:140]: "become" => "will become"
suggest also to remove "only"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Finished 🙂 !
Just in time for press conference of Jens Spahn at 12:00 on phoenix.de !

**Comment by MikeMcC399** [comment]
No surprises in the press conference.
https://www.youtube.com/watch?v=jd5mYXWVaU8

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json:140]: thanks.

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json:139]: thanks.

Comment (MikeMcC399) [reply src/data/faq.json:139]: It still says "incompatible to", not "with".
https://lingohelp.me/preposition-after-adjective/incompatible-with-for-in-to/

I think that "with" is better.

File: src/data/faq_de.json [hunk 3]
```diff
@@ -136,10 +136,10 @@
...
+                            "Diese neuen digitalen Zertifikatsbescheinigungen (die gescannt werden können) werden ab 14. Juni von autorisierten Stellen über einen zentralen Dienst bereitgestellt, der im Auftrag der Bunderegierung entwickelt wurde. Der Feldtest dafür wurde inzwischen beendet, und die Funktion wird nun flächendeckend ausgerollt. Weitere Informationen zu möglichen lokalen Standorten finden Sie hier:  <a href='https://www.aponet.de/artikel/corona-digitaler-impfnachweis-ab-14-juni-in-apotheken-24262' target='_blank' rel='noopener noreferrer'>Corona: Digitaler Impfnachweis ab 14. Juni in Apotheken</a>.",
+                            "Zum Prozess, wie ein Impfzertifikat auf Basis eines Impfnachweises (gelber Impfausweis bzw. der Impfbescheinigung) ausgestellt wird, gibt es  <a href='https://digitaler-impfnachweis-app.apps.public.bfarm.de/impfzertifikat-ausstellen/' target='_blank' rel='noopener noreferrer'>hier</a> weitere Informationen. Das Netz lokaler Aussteller wird stufenweise weiter aufgebaut."
```
Comment (Ein-Tim) [inline src/data/faq_de.json]: Remove one space to much

```suggestion
                            "Zum Prozess, wie ein Impfzertifikat auf Basis eines Impfnachweises (gelber Impfausweis bzw. der Impfbescheinigung) ausgestellt wird, gibt es <a href='https://digitaler-impfnachweis-app.apps.public.bfarm.de/impfzertifikat-ausstellen/' target='_blank' rel='noopener noreferrer'>hier</a> weitere Informationen. Das Netz lokaler Aussteller wird stufenweise weiter aufgebaut."
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
One very small thing. 
Do we want to add this entry to the index: https://www.coronawarn.app/de/faq/#vac_cert_index?

File: src/data/faq_de.json (hunk 3 above)
Comment (dsarkar) [reply src/data/faq_de.json]: thanks will correct later.

Comment (dsarkar) [reply src/data/faq_de.json]: sorry

**Comment by dsarkar** [comment]
@Ein-Tim yes, the index is incomplete. I will update that one as well.

File: src/data/faq_de.json (hunk 3 above)
Comment (Ein-Tim) [reply src/data/faq_de.json]: Will open PR in a few minutes.

**Comment by MikeMcC399** [comment]
@dsarkar 
I have a couple of further grammar corrections for the English language part. Should I do a separate PR?

**Pull request merged**